ocfweb/stats/mirrors.py

```diff
@@ -0,0 +1,42 @@
+from datetime import date
+
+from django.shortcuts import render
+from humanize import naturalsize
+from ocflib.lab.stats import current_semester_start
+from ocflib.lab.stats import get_connection
+
+from ocfweb.caching import periodic
+
+MIRRORS_EPOCH = date(2017, 1, 1)
+
+
+def stats_mirrors(request):
+    return render(
+        request,
+        'stats/mirrors.html',
+        {
+            'title': 'Mirrors Statistics',
+            'bandwidth_semester': bandwidth_semester(),
+            'bandwidth_all_time': bandwidth_all_time(),
+            'start_date': current_semester_start,
+        }
+    )
+
+
+def _bandwidth_by_dist(start):
+    with get_connection() as c:
+        c.execute(
+            'SELECT `dist`, SUM(`up` + `down`) as `bandwidth` FROM `mirrors_public` WHERE `date` > %s'
+            'GROUP BY `dist` ORDER BY `bandwidth` DESC', (start))
+
+    return [(i['dist'], naturalsize(i['bandwidth'])) for i in c]
```

Comment:
can we avoid adding another dependency for this and just write the function ourselves? here's the implementation i usually use:

```
def human_size(bytes):
    for label, n in (
        ('TB', 2**40),
        ('GB', 2**30),
        ('MB', 2**20),
        ('KB', 2**10),
    ):
        if bytes > n:
            return '{:.1f} {}'.format(bytes / n, label)
    else:
        return '{} B'.format(bytes)
```